screener/views.py

```
@@ -746,3 +749,41 @@ def get(self, request, white_label):
+            return Response(
+                {"error": "upgrade, address, and heating_fuel are required."},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        try:
+            client = RewiringAmericaClient()
+            raw = client.fetch_rem_impact(upgrade, address, heating_fuel, water_heater_fuel)
+        except requests.HTTPError as e:
+            try:
+                detail = e.response.json()
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)